src/Components.tsx

```diff
@@ -95,7 +95,7 @@ export const ConnectNotificationBanner = ({
   onNotificationsChange,
 }: {
   collectionOptions?: CollectionOptions;
-  onNotificationsChange?: () => void;
+  onNotificationsChange?: (response?: {total?: number, actionRequired?: number}) => void;
```

Comment:
Why is `response`, `total`, or `actionRequired` optional?

Reply:
I had thought that we should give them the flexibility to pass the params or not, but I can't think of a good reason to keep it as optional. I'm happy to make these required if you think that's best practice.

Reply:
This should match API review, which mentions they are not optional!